src/server/ua_server_worker.c

```diff
@@ -345,6 +350,18 @@ UA_Server_run_startup(UA_Server *server) {
         startMulticastDiscoveryServer(server);
 #endif
 
+    /* create the OverFlowEventType */
+#ifdef UA_ENABLE_EVENTS
+    UA_ObjectTypeAttributes overflowAttr = UA_ObjectTypeAttributes_default;
```

Comment:
Add comment:
The EventQueueOverflowEventType is defined as abstract, therefore we can not create an instance of that type directly, but need to create a subtype. This is already posted on the OPC Foundation bug tracker under the following link for clarification: [insert link here]